app/controllers/sessions_controller.rb

```diff
@@ -1,13 +1,24 @@
 class SessionsController < ApplicationController
   def new
+
   end
 
   def create
-    if user = User.from_omniauth(request.env["omniauth.auth"])
-      session[:user_id] = user.id
-      redirect_to root_path
+    if request.env["omniauth.auth"]
+      if user = User.from_omniauth(request.env["omniauth.auth"])
+        session[:user_id] = user.id
+        redirect_to root_path
+      else
+        redirect_to root_path
+      end
```

Comment:
Do we need the this nesting of 'if/else's' if you route the from for a new user through user or what were you thinking here??

Reply:
it was a copy and paste thing. I actually just saw it right when I put in the PR forgot to refactor